staging/src/k8s.io/apiserver/pkg/util/flowcontrol/request/width.go

```diff
@@ -17,24 +17,42 @@ limitations under the License.
 package request
 
 import (
+	"fmt"
 	"net/http"
+
+	apirequest "k8s.io/apiserver/pkg/endpoints/request"
+	"k8s.io/klog/v2"
+)
+
+const (
+	// the minimum number of seats a request must occupy
+	minimumSeats = 1
+
+	// the maximum number of seats a request can occupy
+	maximumSeats = 10
 )
 
 type WorkEstimate struct {
 	// Seats represents the number of seats associated with this request
 	Seats uint
 }
 
-// DefaultWorkEstimator returns estimation with default number of seats
-// of 1.
-//
-// TODO: when we plumb in actual work estimate handling for different
-//  type of request(s) this function will iterate through a chain
-//  of workEstimator instance(s).
-func DefaultWorkEstimator(_ *http.Request) WorkEstimate {
-	return WorkEstimate{
-		Seats: 1,
+// objectCountGetterFunc represents a function that gets the total
+// number of objects for a given resource.
+type objectCountGetterFunc func(string) (int64, error)
+
+func (f objectCountGetterFunc) Get(key string) (int64, error) {
+	return f(key)
+}
```

Comment:
Why give a func type a method that simply invokes the function?  Wouldn't it be simpler for the callers to just call the func directly?

Reply:
+1 - let's remove it and just call the function directly